public/js/p3/resources/Header.css

```diff
@@ -127,12 +127,16 @@
 
 .P3Header .appLabel {
     font-size: 1em;
-    color: orange;
+    color: #ffbc3e;
     font-weight: 600;
     margin-left: 3px;
     /*margin-bottom: -4px;*/
 }
 
+#widget_globalSearch .dijitPlaceHolder {
+    color: #6f6f6f;
+}
+
```

Comment:
does this match?

Reply:
It does. Dojo generates this ID on the search input tag.

Reply:
thanks for the clarification.